### PR TITLE
Map component for NS local unit

### PR DESCRIPTION
**Pull request by shreeyash07**
## Addresses:
- #823 
- #858 

## Changes
- Update map details in map component in local unit
- Update map details in map tooltip
- Add local unit icons and healthcare icons
- Add Legends in map

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by samshara — APPROVED** [review]

File: app/src/utils/constants.ts
Comment (frozenhelium) [inline app/src/utils/constants.ts:1]: Let's not remove the headings

**Review by frozenhelium — CHANGES_REQUESTED** [review]
Lets remove all the icons

File: app/src/utils/constants.ts
Comment (frozenhelium) [reply app/src/utils/constants.ts:1]: Also remove unused translations and styling

File: app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/common.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export type Validation = 'validated' | 'not_validated';
+
+export const VALIDATED = 'validated' satisfies Validation;
+export const NOT_VALIDATED = 'not_validated' satisfies Validation;
+
+export const TYPE_ADMINISTRATIVE = 1;
```
Comment (frozenhelium) [inline app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/common.ts]: Remove unused constants

File: app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsMap/index.tsx [hunk 2]
```diff
@@ -75,39 +86,151 @@ function emailKeySelector(email: string) {
...
-                    type: localUnit.type.code,
-                },
-            })) : [],
-    }), [localUnitListResponse]);
+    const {
+        response: localUnitMapTooltipResponse,
+    } = useRequest({
+        skip: isNotDefined(clickedPointProperties?.localUnitId),
+        url: '/api/v2/local-units/{id}/',
+        pathVariables: {
```
Comment (frozenhelium) [inline app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsMap/index.tsx]: Let's not use -1 here

File: app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/common.ts (hunk 1 above)
Comment (shreeyash07) [reply app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/common.ts]: no unused constants are present

**Review by frozenhelium — APPROVED** [review]

File: app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/common.ts (hunk 1 above)
Comment (frozenhelium) [reply app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/common.ts]: These constants are not used anywhere

```
export const TYPE_ADMINISTRATIVE = 1;
export const TYPE_HEALTHCARE = 2;
export const TYPE_EMERGENCY_RESPONSE = 3;
export const TYPE_HUMANITARIAN = 4;
export const TYPE_TRAINING = 5;
export const TYPE_OTHER = 6;

export const TYPE_HEALTH_AMBULANCE = 1;
export const TYPE_HEALTH_BLOOD_CENTER = 2;
export const TYPE_HEALTH_HOSPITAL = 3;
export const TYPE_HEALTH_PHARMACY = 4;
export const TYPE_HEALTH_PHC_CENTER = 5;
export const TYPE_HEALTH_RESIDENTIAL = 6;
export const TYPE_HEALTH_TRAINING = 7;
export const TYPE_HEALTH_SPECIAL_SERVICES = 8;
export const TYPE_HEALTH_OTHERS = 9;
```

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**